lib/licenses.nix

```diff
@@ -55,6 +55,12 @@ in mkLicense lset) ({
     fullName = "GNU Affero General Public License v3.0 or later";
   };
 
+  aladdin = {
```

Comment:
Put this modification in a separate commit.

Reply:
Added a separate commit before. Do you have a suggestion about the `aladdin` vs `afpl` naming?

Reply:
SPDX is my source of truth on this case. And they call it Aladdin!

https://spdx.org/licenses/Aladdin.html